doc/manual/manual.tex

```diff
@@ -1339,21 +1354,234 @@ \subsection{Numerical methods}
 gives a detailed description and rationale for the various building blocks.
 
 
-\subsection{Simplifications of the basic equations}
+\subsection{Approximate equations}
+\label{sec:approximate-equations}
+
+There are a number of common variations to equations
+\eqref{eq:stokes-1}--\eqref{eq:temperature} that are frequently used in the
+geosciences. For example, one frequently finds references to the anelastic liquid
+approximation (ALA), truncated anelastic liquid approximation (TALA), and the
+Boussinesq approximation (BA). These can all be derived from the basic
+equations~\eqref{eq:stokes-1}--\eqref{eq:temperature} via various approximations, 
+and we will discuss them in the following. Since they are typically only provided
+considering velocity, pressure and temperature, we will in the following omit
+the dependence on the compositional fields used in previous sections, though
+this dependence can easily be added back into the equations stated below. A
+detailed discussion of the approximations introduced below can also be found in
+\cite{KLKLZTTK10}.
+
+The three approximations mentioned all start by writing the pressure and
+temperature as the sum of a (possibly depth dependent)
+reference state plus a perturbation, i.e., we will write
+\begin{align*}
+  p(\mathbf x,t) &= \bar p(z) + p'(\mathbf x,t),
+  \\
+  T(\mathbf x,t) &= \bar T(z) + T'(\mathbf x,t).
+\end{align*}
+Here, barred quantities are reference states and may depend on the depth $z$
+(not necessarily the third component of $\mathbf x$) whereas primed quantities
+are the spatially and temporally variable deviations of the temperature and
+pressure fields from this reference state. In particular, the reference pressure
+is given by solving the hydrostatic equation,
+\begin{align}
+\label{eq:hydrostatic-pressure}
+  \nabla \bar p = \bar\rho \mathbf g,
+\end{align}
+where $\bar\rho=\rho(\bar p,\bar T)$ is a \textit{reference density} that
+depends on depth and represents a typical change of material parameters and solution 
+variables with depth. $\bar T(z)$ is chosen as an adiabatic profile accounting for the
+fact that the temperature increases as the pressure increases.
```

Comment:
Or do you want to wait until you talk about the individual approximations with this?